html_css/v2/main-course/grid-lessons/positioning-grid-items.md

```diff
@@ -0,0 +1,161 @@
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+Most of our rooms represent a single grid cell. But we have given ourselves a large living room (yay!) We positioned this item using `grid-column-start` and `grid-column-end`. Their property values represent the column grid lines we wish it to start and end with. 
+
+You will also notice we have commented out property values for this item's grid row positioning. Uncomment the `grid-row-start` and `grid-row-end` properties to see how our living room can get even bigger by taking up the grid track between the first and third-row grid lines.
+
+These properties allow us to use our existing grid lines to tell items how many rows and columns each item should span across. Take a minute to play around with the property values here. If the line numbers are confusing, inspect the container using your dev tools to show the grid overlay.
+
+Next, we need to use our space more efficiently. We will make the rest of our rooms span multiple grid cells and fill out the rest of our apartment. 
+
+<p class="codepen" data-height="300" data-default-tab="css,result" data-slug-hash="jOLpmEd" data-editable="true" data-user="dm-murphy" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/dm-murphy/pen/jOLpmEd">
+  Fourth-Positioning-Whole-Apartment</a> by Daniel (<a href="https://codepen.io/dm-murphy">@dm-murphy</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+Now we have the blueprints for our full apartment. If you take a look at the `#kitchen` selector you will see we used shorthand properties here. `grid-column` is just a combination of `grid-column-start` and `grid-column-end` with a slash between the two values. And `grid-row` is the shorthand version for setting an item's row positioning.
+
+One problem with our floor plan is that the bathroom and kitchen are on opposite ends of the apartment. We would save money on the plumbing by placing these two rooms back to back. Take a minute now and see if you can change the starting and ending positions for the bathroom, bedroom and closet so that the bathroom is right next to the kitchen. You can use either the long or shorthand properties here.
+
+### `grid-area`
+
+You now know how to position your grid items using row and column lines. But there are other ways to position items and this is where things can get a little confusing. 
+
+There is an even shorter shorthand for positioning grid items with start and end lines. You can combine `grid-row-start` / `grid-column-start` / `grid-row-end` / `grid-column-end` into one line using `grid-area`.
+
+Our living room above can be written out like this:
+
+~~~css
+/* styles.css */
+
+#living-room {
+  grid-area: 1 / 1 / 3 / 6
```

Comment:
needs a semi-colon

Reply:
How did I do this twice! :-( Thanks for catching these!